sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
-## 7.3.0-beta.3 (Unreleased)
+    
+## 7.3.0 (2021-06-25)
```

Comment:
We now use "Features Added" instead of "New Features". Refer to https://github.com/Azure/azure-sdk/blob/main/docs/policies/releases.md#change-logs 

We may need to add "Dependency Updates" because at least azure-core-amqp dependency is changed.